.github/workflows/build.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       working-directory: ./bridge
     - run: bash <(curl -s https://codecov.io/bash)
       name: codecov
-      working-directory: ./bridge
+      working-directory: ./bridge/test
```

Comment:
Is it temporary?

Reply:
No, `jest` created coverage result at `/bridge/test/coverage` and I changed the `working-directory` to match.

Reply:
But I found the configuration to move the directory's path so I will revert it.